packages/emitter-framework/src/python/components/import-statement.tsx

```diff
@@ -0,0 +1,21 @@
+import { mapJoin, memo } from "@alloy-js/core";
+
+export interface ImportSymbol {
+  package: string;
+  name?: string;
+  wildcard?: boolean;
+}
+
+export interface ImportStatementsProps {
+  imports: ImportSymbol[];
+}
+
+export function ImportStatements(props: ImportStatementsProps) {
+  return memo(() =>
+    mapJoin(props.imports, (importProp) =>
+      () => <ImportStatement {...importProp} />));
+}
+
+export function ImportStatement(props: ImportSymbol) {
```

Comment:
This is fine for now but eventually will need to create symbols for the local bindings such that they can be referenced and name disambiguated.

Reply:
The TypeScript implementation also allows more granular updates of imports, it looks like this implementation depends on `props.imports` being updated whenever a new import is added, which would mean reconstructing all import statements.